README.md

```diff
@@ -42,7 +42,22 @@ The project has two main components:
 
 * Support for running distributed workers.
 
-## Running and tests
+## Running
+
+To run the project, certain environment variables need to be set. These variables can be found in the [.env](.env) file.
+
+Each worker, when running, fetches mosaic settings from an API. A mock API is available, detailed in, detailed in [tasks.json](testing/tasks.json). Below is a breakdown of each field in the JSON:
```

Comment:
@mauricioabreu,

In `A mock API is available, detailed in, detailed in ...` is repeated twice times, it's correct?